docs/upgrade/11_to_12/workflows_mariadb.py

```diff
@@ -17,6 +17,7 @@
 user = "opencast"
 password = "dbpassword"
 host = "127.0.0.1"
+port = 3306
```

Comment:
Adding default database port numbers is unrelated to what you said you want to achieve with this PR. I think it would be good practice to submit a seperate PR for it.

Reply:
I discussed this with @gregorydlogan on technical meeting of OC, and we agreed to keep it in.